src/virtualship/cli/_fetch.py

```diff
@@ -145,8 +145,8 @@ def _fetch(path: str | Path, username: str | None, password: str | None) -> None
         complete_download(download_folder)
         click.echo("Ship data download based on space-time region completed.")
 
-    if "DRIFTER" in instruments_in_schedule:
-        print("Drifter data will be downloaded")
+    if "DRIFTER" in str(instruments_in_schedule):
```

Comment:
Just wondering what the motivation for this is? Was it not working before?

Since `instruments_in_schedule` is a call to `schedule.get_instruments()` giving a `set[InstrumentType]`, I think the right way to do this would be

```suggestion
    if InstrumentType.DRIFTER in instruments_in_schedule:
```
(also making sure `InstrumentType` is imported at the top)

Reply:
Yes, much more sensible. I've made these changes. I've added the `InstrumentType` import within the `_fetch` function to avoid circular import errors. Does that seem sensible?

Reply:
Yes, that sounds good. I'm not sure why we're getting circular imports, might be worth me looking at the package structure.

But yes, a lot of code sometimes puts the import in the function body rather than the top of the file